Print_A_Bot/controls/consumers.py

```diff
@@ -0,0 +1,18 @@
+from django.http import HttpResponse
```

Comment:
again pretty much following https://channels.readthedocs.io/en/stable/getting-started.html

this piece is gonna change heavily as it starts to use the utils to control the robot